integrations/src/test/java/io/jenkins/plugins/casc/ViewJobFiltersTest.java

```diff
@@ -0,0 +1,66 @@
+package io.jenkins.plugins.casc;
+
+import hudson.model.Descriptor;
+import hudson.model.ListView;
+import hudson.model.View;
+import hudson.util.DescribableList;
+import hudson.views.BuildDurationFilter;
+import hudson.views.BuildStatusFilter;
+import hudson.views.SecurityFilter;
+import hudson.views.ViewJobFilter;
+import io.jenkins.plugins.casc.misc.ConfiguredWithReadme;
+import io.jenkins.plugins.casc.misc.JenkinsConfiguredWithReadmeRule;
+import java.util.Collection;
+import jenkins.model.Jenkins;
+import org.junit.Rule;
+import org.junit.Test;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+public class ViewJobFiltersTest {
+
+    @Rule
+    public JenkinsConfiguredWithReadmeRule j = new JenkinsConfiguredWithReadmeRule();
+
+    @Test
+    @ConfiguredWithReadme(value = "view-job-filters/README.md")
+    public void configure_view_job_filters() throws Exception {
+        BuildDurationFilter duration = null;
+        BuildStatusFilter status = null;
+        SecurityFilter security = null;
+
+        Jenkins jenkins = j.jenkins;
+        Collection<View> views = jenkins.getViews();
+        for (View view : views) {
```

Comment:
better to just grab `views.get(0)`, you know how many elements are in the list, and then remove the for loop

Reply:
so isn't it a problem that the [compilation failed](https://ci.jenkins.io/blue/organizations/jenkins/Plugins%2Fconfiguration-as-code-plugin/detail/PR-1603/30/pipeline) with the error _cannot find symbol:   method get(int)_?